### DISCUSSION TITLE
i swear i could change overhang perimiter detection but i lost it again.

### DISCUSSION THREAD

**Discussion by levleon13** [supermerill/SuperSlicer#2471] [Q&A]
i've been told an unclear answer about how this is not directly possible to change ( no such parameter to change ) but a few weeks ago i found a solution and i could acheve better cooling of my melty PETG which was FKN awesome. But i have changed something in the settings that brought me back to very high angle detection only. i remember that i could control it ( i inputed different values and seen diffrent angle detection, BUT I CANNOT REMMEBER WHAT I DID !!! and i am in panic, because this is one of the single soultions for my petg prints to look good ).
YELP?

(BLUE AREA)
![image](https://user-images.githubusercontent.com/77578163/156015660-748ab7d5-a95d-44bd-90ba-4a978fbf0a3f.png)
i mean, how is that not an overhang hehe??
![image](https://user-images.githubusercontent.com/77578163/156018827-037c1e56-46ad-4f73-81ad-744d64a7e77a.png)

**Comment by neophyl** [comment]
Print Settings>Perimeters & Shell>Overhangs  Thresholds for Bridge Speed and Fan and Bridge flow.  You have to be in expert mode to see the settings.
